servers/src/mining/mine_block.rs

```diff
@@ -259,7 +259,7 @@ fn get_coinbase(
 /// Will retry based on default "retry forever with backoff" behavior.
 fn create_coinbase(dest: &str, block_fees: &BlockFees) -> Result<CbData, Error> {
 	let url = format!("{}/v1/wallet/foreign/build_coinbase", dest);
-	match api::client::post(&url, None, &block_fees) {
+	match api::client::GrinClient::default().post(&url, &block_fees) {
```

Comment:
I know that you are not changing anything here but how does this call work if the wallet uses auth?

Reply:
I have the same question actually